folio.lua

```diff
@@ -163,11 +166,30 @@ function key(n, z)
         end
         page = 'scripts'
         redraw()
+      elseif s:sub(1, #'url: ') == 'url: ' then
+        local url_name = s:sub(#'url: ' + 1, #s)
+            clock.run(function()
+	          print(url_name)
+		  pull(url_name)
+	        end)
+        redraw()
       end
     end
   end
 end
 
+function pull(url)
```

Comment:
Do we need `mkdir` and `sudo`? I would think `git clone` will create the target directory.

Reply:
a first test without creating the folder manually did not work. Later I also had to add administrator access ... so maybe using "sudo" you don't need to create the folder manually ... did you do a test?